recipes/oso/meta.yaml

```diff
@@ -39,11 +39,10 @@ test:
   imports:
     - oso
     - polar
-    - sqlalchemy
   commands:
     - cd languages/python/oso
     - pip check
-    - pytest -vv tests -k "not test_parser_errors"
+    - pytest -vv tests -k "not test_parser_errors and not test_repr_when_logging"
```

Comment:
Now that basic testing is established, consider adding coverage:

```bash
pytest -vv tests --cov=oso --cov=polar --no-cov-on-fail --cov-report=term-missing:skip-covered --cov-fail-under=100
```

This can help find serious regressions (e.g. missing test dependencies). Alternately: remove `pytest-cov` (and any other unused test dependencies): e.g. is `sqlalchemy` really needed? This is important, as sometimes spurious heavyweight test deps bring along transient deps that shadow real runtime deps.

Reply:
fail under 100? cov is at 85.62%. should I just drop to like "fail under 80"?